cmd/snap-update-ns/utils_test.go

```diff
@@ -52,6 +53,35 @@ func (s *utilsSuite) SetUpTest(c *C) {
 	s.BaseTest.AddCleanup(restore)
 	s.log = buf
 	s.as = &update.Assumptions{}
+
```

Comment:
Similarly to `change_test.go`, should these return the errors as well to stop the test early?